.github/workflows/main.yml

```diff
@@ -73,7 +73,19 @@ jobs:
       docs: ${{ steps.include.outputs.docs }}
 
     steps:
-        # This function must be inlined in main.yml, or we'd be forced to checkout the repo
+      - name: 'Checkout the scripts'
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            .github
```

Comment:
I'm guessing this does not add much extra time?

Reply:
Yup! This checks out _only these files_, not the whole JDK workspace. Seems to run in < 1s. This would also resolve a TODO to move the rest of `prepare` job to scripts: it was apparently inlined _because_ we did not want to check out the entire JDK repo.

Reply:
Yes, that was my doing. I did not know about, or think about, the ability to check out just part of the tree. This might open for some cleanup and better structuring of the code...